modules/key-distributor/cfn/encrypter/function.go

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
We should probably put this in a `scripts/` directory.

Reply:
Not sure I agree that a Go file that gets packaged into a Cloud Function belongs in a `scripts` directory. Is this a pattern elsewhere?

Reply:
We generally try to place any scripts that Terraform references during execution in a well-known directory. In this case, a `files/` directory would actually be appropriate since we're not executing the script.